### ISSUE TITLE
Add entity flags

### ISSUE THREAD

**Issue by Matthias1590** [IntellectualSites/PlotSquared#3813]
### What feature do you want to see added?

A flag that can be set per plot which can be used to disallow certain entities on that plot, this would be useful to prevent people from spamming entities to create lag without having to disable entities on the server entirely

### Are there any alternatives?

Not that I know of, other plugins could definitely get rid of entities in certain positions but they wouldn't work with the plot system

**Comment by SirYwell** [comment]
See <https://intellectualsites.github.io/plotsquared-documentation/plot-flags.html#_integer_2> and <https://intellectualsites.github.io/plotsquared-documentation/plot-flags.html#_entity_categories>. That should be enough to prevent malicious behavior.

**Comment by Matthias1590** [comment]
To clear up any confusion, I am aware of the entity cap flag, this flag would work similar to that except it would take a pattern (much like brigadier does), for example @e[type=!player] which would disallow any entity apart from players

**Comment by kryllyxofficial01** [comment]
Would this be what you're looking for?
https://intellectualsites.github.io/plotsquared-documentation/commands.html#_setflag

**Comment by Matthias1590** [comment]
> Would this be what you're looking for?
> https://intellectualsites.github.io/plotsquared-documentation/commands.html#_setflag

Not really, I'm aware of the flags system, I'm asking for one to be added

**Comment by kryllyxofficial01** [comment]
> > Would this be what you're looking for?
> > https://intellectualsites.github.io/plotsquared-documentation/commands.html#_setflag
> 
> Not really, I'm aware of the flags system, I'm asking for one to be added

Ah, I see
